css-view-transitions-1/Overview.bs

```diff
@@ -497,6 +499,19 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 	This may be confusing, since the elements themselves are not necessarily referring to the same object,
 	but it is a useful model to consider them to be visual states of the same conceptual page entity, that we happen to call "element".
 
+# Layout and rendering changes # {#layout-rendering-changes}
+
+	Both {{Element}}s and [=pseudo-elements=] have a <dfn>involved in a view transition</dfn>, a boolean. Initially false.
```

Comment:
```suggestion
	Both {{Element}}s and [=pseudo-elements=] have an <dfn>involved in a view transition</dfn> boolean, initially false.
```

Just fixing a/an agreement and making the text more natural with the shape of the term.